src/tests/unit/auto/get_device_list.cpp

```diff
@@ -95,20 +94,21 @@ TEST_P(GetDeviceListTest, GetDeviceListTestWithExcludeList) {
 // ConfigParams {devicePriority, expect metaDevices, ifThrowException}
 
 const std::vector<ConfigParams> testConfigs = {
-    //
-    ConfigParams {"CPU,GPU,VPUX",
-        "CPU,GPU,VPUX"},
-    ConfigParams {"VPUX,GPU,CPU,-GPU.0",
-        "VPUX,GPU.1,CPU"},
-    ConfigParams {"-GPU.0,GPU,CPU",
-        "GPU.1,CPU"},
-    ConfigParams {"-GPU.0,GPU",
-        "GPU.1"},
-    ConfigParams {"-GPU.0", "CPU,GPU.1,VPUX"},
-    ConfigParams {"-GPU.0,-GPU.1", "CPU,VPUX"},
-    ConfigParams {"-GPU.0,-CPU", "GPU.1,VPUX"}
-};
-
+    ConfigParams{" ", " "},
```

Comment:
where will the exception happen for this case?

Reply:
No exception happened here as this test case is just to parse out the valid device from the device priorities rather than is to check if loadNetwork() works. Following the current logic of Auto::GetDevicelist(), device with name " " will be parsed out as vaild device because it was specified by user.